services/workspace_tabs.py

```
@@ -0,0 +1,497 @@
+
+        # Load bubbles
+        for row in global_db.execute("SELECT key, value FROM cursorDiskKV WHERE key LIKE 'bubbleId:%'"):
+            parts = row["key"].split(":")
+            if len(parts) >= 3:
+                bid = parts[2]
+                try:
+                    b = json.loads(row["value"])
+                    if isinstance(b, dict):
+                        bubble_map[bid] = b
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)